test/test_browser.py

```diff
@@ -4655,20 +4685,28 @@ def test_webgl_simple_enable_extensions(self):
                '-sGL_SUPPORT_SIMPLE_ENABLE_EXTENSIONS=' + str(simple_enable_extensions)]
         self.btest_exit('webgl2_simple_enable_extensions.c', args=cmd)
 
-  @parameterized({
-    '': ([],),
-    'closure': (['-sASSERTIONS', '--closure=1'],),
-    'main_module': (['-sMAIN_MODULE=1'],),
-  })
+  @also_with_wasm64
+  @requires_graphics_hardware
+  def test_webgpu_basic_rendering(self):
+    self.btest_exit('webgpu_basic_rendering.cpp', args=['-sUSE_WEBGPU'])
+
+  @also_with_wasm64
   @requires_graphics_hardware
-  def test_webgpu_basic_rendering(self, args):
-    self.btest_exit('webgpu_basic_rendering.cpp', args=['-sUSE_WEBGPU'] + args)
+  def test_webgpu_basic_rendering_closure(self):
+    self.btest_exit('webgpu_basic_rendering.cpp', args=['-sUSE_WEBGPU', '-sASSERTIONS=1', '--closure=1'])
 
+  @also_with_wasm64
+  @requires_graphics_hardware
+  def test_webgpu_basic_rendering_main_module(self):
+    self.btest_exit('webgpu_basic_rendering.cpp', args=['-sUSE_WEBGPU', '-sMAIN_MODULE=1'])
```

Comment:
Can you revert the splitting of this test into 3?  I think we have enough new `@also_with_wasm64` that we can land this change with this.

We can look at other ways to inject `wasm64` perhaps into all browser tests as a followup?

Reply:
Reverted.

+1 for having another way to run the tests with wasm64 because chaining decorators doesn't seem to foward the parameterized arguments, so I've had to split the tests by hand.